.github/workflows/Repology.yml

```diff
@@ -39,13 +39,11 @@ jobs:
           author: "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
           commit-message: "Update Repology JSON"
           delete-branch: true
-          team-reviewers: chromebrew/active
       - name: Trigger Unit Test Workflow & Add Reviewer Team
         env:
           GH_TOKEN: ${{ secrets.CREW_PR_TOKEN }}
           PR_NUMBER: :${{ steps.cpr.outputs.pull-request-number }}
         run: |
             echo "Flipping pr ${PR_NUMBER} state."
             gh pr close "${PR_NUMBER}" && gh pr reopen "${PR_NUMBER}"
-            gh pr edit "${PR_NUMBER}" --add-reviewer chromebrew/active
```

Comment:
Why remove this? It appears to work?

Reply:
See https://github.com/chromebrew/chromebrew/actions/runs/14429303553, our secret token does not have enough permission to do it

Reply:
Let's try switching `GH_TOKEN: ${{ secrets.CREW_PR_TOKEN }}` to `GH_TOKEN: ${{ secrets.CREW_PR_OPS_TOKEN }}` to see if that fixes things?

Reply:
`GH_TOKEN: ${{ secrets.CREW_PR_TOKEN }}` doesn't make a difference here, same error :/